content/guides/models/registry/search_registry.md

```diff
@@ -7,13 +7,89 @@ title: Find registry items
 weight: 7
 --- 
 
-Use the global search bar in the W&B Registry App to find a specific registry, collection, artifact version tag, collection tag, or alias. To search globally:
+Use the global search bar in the W&B Registry App or Python SDK to find a specific registry, collection, collection tag, artifact version name, artifact tag, or artifact alias.
```

Comment:
This makes it sound like the Python SDK has a global search bar? What about something like "To find a specific...., you can use the [global search bar] in the App or you can [search programmatically using the Python SDK].